src/sentry/tasks/store.py

```diff
@@ -293,6 +293,11 @@ def save_event(cache_key=None, data=None, start_time=None, event_id=None, **kwar
     try:
         manager = EventManager(data)
         manager.save(project)
+    except HashDiscarded as exc:
```

Comment:
At some point we're probably going to want to record this happening here as compared to the pre-processing filters so that we can identify the hit rate of those filters. I don't know that this needs to necessarily be part of this patch but it needs to happen at some point so we should make sure that a todo is recorded somewhere